### PR TITLE
Makes ventcrawling code less godawful

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

this particular snippet of code was so incredibly bad that it made me webedit PR, that's how bad it is

## Why It's Good For The Game

if you have defines use them. if you know it's at least 1, why are you checking if it's exactly 1 by checking if it's less than 2. that's not clever, that's bad programming. why are you using a FOR LOOP to check if THERE'S ANYTHING IN A LIST. THE LINE RIGHT ABOVE USED .LEN. PLEASE.

## Changelog
:cl:
code: made ventcrawling not intentionally obfuscated badcode
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Driven by pure hatred, I love it.

**Comment by itseasytosee** [comment]
Man, we really need a web edit tag.

**Comment by Jared-Fogle** [comment]
We do, we don't apply "Verified" on people who setup GPG keys 😛

File: code/modules/mob/living/ventcrawling.dm [hunk 1]
```diff
@@ -55,15 +55,9 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, typecacheof(list(
...
-				var/failed = 0
+			if(iscarbon(src) && ventcrawler == VENTCRAWLER_NUDE)
 				var/list/items_list = get_equipped_items(include_pockets = TRUE)
-				if(items_list.len)
-					failed = 1
-				for(var/obj/item/I in held_items)
-					failed = 1
-					break
-				if(failed)
+				if(items_list.len || held_items.len)
```
Comment (AnturK) [inline code/modules/mob/living/ventcrawling.dm:60]: This is incorrect held items is always length of available hands.

**Review by AnturK — COMMENTED** [review]
Too late

File: code/modules/mob/living/ventcrawling.dm (hunk 1 above)
Comment (Putnam3145) [reply code/modules/mob/living/ventcrawling.dm:60]: THEN WHY WAS THIS MERGED OH GOD

Comment (Putnam3145) [reply code/modules/mob/living/ventcrawling.dm:60]: this was already fixed upstream and i am no longer panicking

**Pull request merged**